### PR TITLE
(feat) Extend the patient object to support `other` and `unknown` genders

### PR DESCRIPTION
**Pull request by VivekAlladi**
## Requirements

- [x] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [ ] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide).
- [ ] My work includes tests or is validated by existing tests.

## Summary

I'm working on displaying Pictograms on Ampath forms based on the gender of the patient. Currently, I'm able to render options for male and female patients based on `model['sex'] = 'M' or 'F'`. 

I have now slightly modified the model object so that now it also stores `O` for `Other` and `U` for `Unknown` genders. 

[Issue](https://tfs.ext.icrc.org/ICRCCollection/Pearlii/_sprints/backlog/ThoughtWorks/Pearlii/Application/EMR/ThoughtWorks/2023/2023%20Sprint%205?workitem=349498)

### CODE REVIEW

File: packages/esm-form-entry-app/src/app/form-data-source/form-data-source.service.ts [hunk 1]
```diff
@@ -303,7 +303,21 @@ export class FormDataSourceService {
...
+        case 'male':
+          return 'M';
+        case 'female':
+          return 'F';
+        case 'other':
+          return 'O';
+        case 'unknown':
+          return 'U';
+        default:
+          return 'M';
```
Comment (jnsereko) [inline packages/esm-form-entry-app/src/app/form-data-source/form-data-source.service.ts:318]: Why is the default gender Male? Realistically, you can't an individual's gender until he/she tell you.

Comment (jnsereko) [inline packages/esm-form-entry-app/src/app/form-data-source/form-data-source.service.ts:318]:
```suggestion
          return 'U';
```

**Review by jnsereko — CHANGES_REQUESTED** [review]
Why is the default gender Male? Realistically, you can't an individual's gender until he/she tell you.

**Review by denniskigen — APPROVED** [review]

**Review by denniskigen — APPROVED** [review]
Thanks, @VivekAlladi.

**Pull request merged**